script/build.go

```diff
@@ -139,8 +145,8 @@ func buildCommand(dir, buildos, buildarch string) error {
 
 	args := make([]string, 1, 6)
 	args[0] = "build"
-	if len(LdFlag) > 0 {
-		args = append(args, "-ldflags", LdFlag)
+	if len(LdFlags) > 0 {
+		args = append(args, "-ldflags", strings.Join(LdFlags, " "))
```

Comment:
I don't think this is quite how the `-ldflags` flag works in practice. The `-ldflags` expects a string rather than introducing a set of arguments. Applying this change yields an empty ldflags set, and instead tries to pass the contents of LdFlags as packages for `go build` to build.

```
~/g/git-lfs (no-dwarf-tables!) $ ./script/bootstrap && gzip < bin/git-lfs | wc -c
Using go1.8rc3
can't load package: package github.com/git-lfs/git-lfs/config.GitCommit=5f183083: cannot find package "github.com/git-lfs/git-lfs/config.GitCommit=5f183083" in any of:
        /usr/local/Cellar/go/1.8rc3/libexec/src/github.com/git-lfs/git-lfs/config.GitCommit=5f183083 (from $GOROOT)
        /Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/config.GitCommit=5f183083 (from $GOPATH)
can't load package: package -s: cannot find package "-s" in any of:
        /usr/local/Cellar/go/1.8rc3/libexec/src/-s (from $GOROOT)
        /Users/ttaylorr/go/src/-s (from $GOPATH)
can't load package: package -w: cannot find package "-w" in any of:
        /usr/local/Cellar/go/1.8rc3/libexec/src/-w (from $GOROOT)
        /Users/ttaylorr/go/src/-w (from $GOPATH)
can't load package: package -o: cannot find package "-o" in any of:
        /usr/local/Cellar/go/1.8rc3/libexec/src/-o (from $GOROOT)
        /Users/ttaylorr/go/src/-o (from $GOPATH)
can't load package: package bin/git-lfs: cannot find package "bin/git-lfs" in any of:
        /usr/local/Cellar/go/1.8rc3/libexec/src/bin/git-lfs (from $GOROOT)
        /Users/ttaylorr/go/src/bin/git-lfs (from $GOPATH)

2017/02/14 10:02:12 exit status 1
exit status 1
```